Formula/percona-server.rb

```diff
@@ -55,6 +56,10 @@ def install
       "COMMAND /usr/bin/libtool -static -o ${TARGET_LOCATION}",
       "COMMAND libtool -static -o ${TARGET_LOCATION}"
 
+    inreplace "plugin/percona-pam-for-mysql/src/dialog.c",
+      "#include <stdlib.h>",
+      "#include <stdlib.h>\n#include <../strings/my_stpnmov.c>"
```

Comment:
What is the bug you are fixing? Has it been reported upstream and are they integrating the patch there? Add a comment giving link and saying that the `inreplace` can be removed for next version.